CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+# 84.3.0
+
+This minor release adds dynamic type support to Chips, new theming extensions for Lists, updates to Bottom Drawer elevation shadow. It also includes minior bug fixes and documentation updates to multiple components.
+
+## New features
+
+# Dynamic type support
+
+'MDCChipView' now support Dynamic font type.
```

Comment:
Replace the `'` with a back-tick (`) in order to make it appear as code.

Reply:
Done